modules/default.nix

```diff
@@ -16,6 +16,7 @@ let
     ./misc/gtk.nix
     ./misc/news.nix
     ./misc/pam.nix
+    ./config/fonts.nix
```

Comment:
I think the `fonts.nix` file could go into the `misc` directory rather than a new `config` directory.